packages/wallet/.eslintrc.js

```diff
@@ -0,0 +1,241 @@
+const rules = {
```

Comment:
Many of these rules seem to be duplicates of rules in other packages. Can we consolidate them in root `.eslintrc.js` file please?

Reply:
This is the plan once all packages are using ESLint

Reply:
It doesn't seem necessary that all packages use ESLint. Those which use ESLint can extend the root `.eslintrc.js`; those which don't, would not.

Reply:
Why does the plan need to blocked on that? All you need are two packages using `eslint` for there to warrant a root level `.eslintrc.js` file.